CODE_STYLE.md

```diff
@@ -5,17 +5,15 @@ considerations.
 
```

Comment:
On line 3, "We follow the standard Go style using gofmt"

May be better to indicate that we actually use `goimports`, but it isn't a big problem.